ora2/jobs/AddCourseToSandbox.groovy

```diff
@@ -0,0 +1,56 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    * folder-name: some-folder
+    * credential : your-ssh-credential-name
+    * email : e-mail-contact@example.com
```

Comment:
Can you use the style of var we've used in other jobs.  We tend towards ALL_CAPS.  Please also name credential something more specific (SSH_KEY).  Similarly for email - something like contact email?